src/css/components/_captions-settings.scss

```diff
@@ -26,30 +26,28 @@
   .vjs-text-track-settings .vjs-modal-dialog-content {
     display: grid;
     grid-template-columns: 1fr 1fr;
-    grid-template-rows: 1fr auto;
+    grid-template-rows: 1fr;
+    min-height: 23em;
```

Comment:
We set the min-height here so that the `settings-controls` will never touch the bottom of the modal.